pyxnat/core/interfaces.py

```diff
@@ -332,20 +333,37 @@ def _exec(self, uri, method='GET', body=None, headers=None, force_preemptive_aut
 
         response = None
 
-        if method is 'PUT':
-            response = self._http.put(uri, headers=headers, data=body, **kwargs)
-        elif method is 'GET':
-            response = self._http.get(uri, headers=headers, params=body, **kwargs)
-        elif method is 'POST':
-            response = self._http.post(uri, headers=headers, data=body, **kwargs)
-        elif method is 'DELETE':
-            response = self._http.delete(uri, headers=headers, data=body, **kwargs)
-        elif method is 'HEAD':
-            response = self._http.head(uri, headers=headers, data=body, **kwargs)
-        else:
-            print('Unsupported HTTP method (%s)'%method)
+        def query(method, uri, headers, body, kwargs):
```

Comment:
Consider calling this function `request`

Reply:
Fixed in 9dfde5c.